app/models.py

```diff
@@ -0,0 +1,11 @@
+
+class User():
+    id = 0
```

Comment:
Look at the suggestion of using a dataclass here. In dataclasses you can assign default value and it will assign default value at instance level not class level. Moreover dataclass will create an `__init__` method for you.
(Just be wary of reference types like list or dict, their default construction will assign a reference instead of value so all instances will hold the same reference, so the same list so do not do this:

```python
@dataclass
class Test:
     a: int = 1 # thats fine, int is a value type
     b: List[int] = [] # that is not fine, list is a reference type 
```

instead you could do this
```python
@dataclass
class Test:
    a: List[int] = field(default_facory=list)
```

The same applies to a function arguments so do not do this
```python
def foo(a = []):
    a.append(1)
    return a
```

It will reuse the default list so multiple calls would result in:

```
>>> foo()
[1]
>>> foo()
[1, 1]
>>> foo()
[1, 1, 1]
```
instead assign `None` as the default value and check so:

```python
def foo(a = None):
    if a is None:
        a = []
    a.append(1)
    return a
```

This will work as intended